pkgs/development/python-modules/quart/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, pythonOlder
+, click
+, aiofiles
+, werkzeug
+, jinja2
+, hypercorn
+, itsdangerous
+, blinker
+, hypothesis
+, pytestCheckHook
+, pytest-asyncio
+, poetry-core
+}:
+
+buildPythonPackage rec {
+  pname = "quart";
+  version = "0.17.0";
+  disabled = pythonOlder "3.7";
+  format = "pyproject";
+
+  src = fetchFromGitLab {
+    owner = "pgjones";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-xpMvCOFoSpuit+KnLs3iWwneK++mLgjD7E6vTUUMwaU=";
+  };
+
+  prePatch = ''
+    substituteInPlace pyproject.toml \
+        --replace "--no-cov-on-fail" ""
+  '';
+
+  propagatedBuildInputs = [
+    click
+    aiofiles
+    jinja2
+    hypercorn
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    poetry-core
+    blinker
+    werkzeug
+    hypothesis
+    itsdangerous
+    pytest-asyncio
+  ];
+
+  pythonImportsCheck = [ "quart" ];
+
+  meta = with lib; {
+    homepage = "https://pgjones.gitlab.io/quart/";
```

Comment:
Seems like upstream just moved: https://github.com/pallets/quart

Reply:
As @pbsds mentioned, we can move this PR to 0.18.0. It requires `Werkzeug >= 2.2.0`, but that is [already merged to staging](https://github.com/NixOS/nixpkgs/pull/187000), so I think it shouldn't be an issue if we make this PR target the `staging` branch.